paper/paper.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'DeepRank2: Mining 3D Protein Structures with Geometric Deep Learning'
+tags:
+  - Python
+  - PyTorch
+  - structural biology
+  - geometric deep learning
+  - 3D protein structures
+  - protein-protein interfaces
+  - single-residue variants
+authors:
+  - name: Giulia Crocioni
+    orcid: 0000-0002-0823-0121
+    corresponding: true
+    affiliation: 1
+    equal-contrib: true
+  - name: Dani L. Bodor
+    orcid: 0000-0003-2109-2349
+    affiliation: 1
+    equal-contrib: true
+  - name: Coos Baakman
+    orcid: 0000-0003-4317-1566
+    affiliation: 2
+    equal-contrib: true
+  - name: Daniel Rademaker
+    orcid: 0000-0003-1959-1317
+    affiliation: 2
+  - name: Dario Marzella
+    orcid: 0000-0002-0043-3055
+    affiliation: 2
+  - name: Gayatri Ramakrishnan
+    orcid: 0000-0001-8203-2783
+    affiliation: 2
+  - name: Sven van der Burg
+    orcid: 0000-0003-1250-6968
+    affiliation: 1
+  - name: Farzaneh Meimandi Parizi
```

Comment:
Farzaneh M. Parizi

Also please move her name higher in the author list, Li and I would suggest just after Coos or Gayatri's name.